src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -55,7 +62,7 @@ def test_device_desc(self):
         self.assertEqual(output.strip(), 'ACS-MSN2700')
 
     def test_device_desc_mgmt_ip(self):
-        argument = '-v "MGMT_INTERFACE.keys()[0]" -M "' + self.sample_device_desc + '"'
+        argument = '-v "(MGMT_INTERFACE.keys()|list)[0]" -M "' + self.sample_device_desc + '"'
```

Comment:
Why do you need list here?

Reply:
In Python 3, Jinja2 template will return dict_keys iterable object (and not index-able) and so in order to maintain the existing behavior, we cast it into list.